problemi/boltzmannDistribution/include/boltzmann/boltzmann.h

```diff
@@ -0,0 +1,33 @@
+#ifndef BOLTZMANN_H
+#define BOLTZMANN_H
+
+#include <random>
+#include <string>
+#include <vector>
+
+struct SimStats {
+  double mean;
+  double stddev;
+};
+
+class BoltzmannSimulator {
+ public:
+  BoltzmannSimulator(int molecules, double kBT = 2.0, double hparam = 10.0,
+                     unsigned seed = 0);
+
+  void run(int nInteractions);
+  const std::vector<double>& getEnergies() const { return energies_; }
```

Comment:
in tutti gli esempi del corso e negli esercizi usiamo sempre snake_case per funzioni e variabili, non camelCase.